packages/cfpb-design-system/src/components/cfpb-buttons/button-group.scss

```diff
@@ -5,4 +5,5 @@
   display: flex;
   gap: math.div(10px, $base-font-size-px) + rem;
   flex-wrap: wrap;
+  align-items: center;
```

Comment:
We could also consider `align-items: baseline` for this, so that the button text and link text bottoms are aligned with each other.

Reply:
IMHO `baseline` looks better